README.md

```diff
@@ -76,6 +76,103 @@ public void handleJobFoo(final JobClient client, final ActivatedJob job) {
 }
 ```
 
+### Fetch Variables
+
+You can access all variables of a process via the job:
+
+```
+@ZeebeWorker(type = "foo")
+public void handleJobFoo(final JobClient client, final ActivatedJob job) {
+  String variable1 = (String)job.getVariablesAsMap().get("variable1");
+  sysout(variable1);
```

Comment:
It is not a thing, this way this would require a method with this name. I doubt that code will really be used - but I am also happy to change it to System.outprintln 👍